cmds/boot/fitboot/main.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2017-2019 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"flag"
+	"log"
+	"os"
+
+	"github.com/u-root/u-root/pkg/boot/fit"
+)
+
+var (
+	dryRun    = flag.Bool("dryrun", true, "Do not actually kexec into the boot config")
+	debug     = flag.Bool("d", true, "Print debug output")
+	rootfs    = flag.String("r", "", "Root file system name")
+	cmdline   = flag.String("c", "earlyprintk=ttyS0,115200,keep console=ttyS0", "command line")
+	initramfs = flag.String("i", "", "InitRAMFS node name -- default none")
+)
+
+var v = func(string, ...interface{}) {}
+
+func main() {
+	flag.Parse()
+	if *debug {
+		v = log.Printf
+	}
+	if len(flag.Args()) < 2 {
+		log.Fatal("Usage: fitboot <file> <node name>")
```

Comment:
Could you somehow make "<node name>" default to the first kernel?

Reply:
Next round if that's ok :-)

but not like that. I expect we won't need to name the kernel.